torch/csrc/jit/codegen/cuda/iter_visitor.cpp

```diff
@@ -53,7 +53,7 @@ void IterVisitor::traverseFrom(
   FusionGuard fg(fusion);
   std::unordered_set<Statement*> visited;
   stmt_stack.clear();
-  stmt_stack.emplace_back(from.rbegin(), from.rend());
+  stmt_stack.emplace_back(std::vector<Statement*>(from.rbegin(), from.rend()));
```

Comment:
no we don't need `std::vector<Statement*>` here.

Reply:
Fixed.